### PR TITLE
Add defs for prosemirror-model v1.x.x

### PR DESCRIPTION
**Pull request by rpowelll**
This probably needs more exhaustive tests, but I've been using these definitions on a moderately complex side project for a little bit now and I'm finally happy enough with them to open a PR. Stay tuned for PRs for `prosemirror-state`, `prosemirror-view`, etc.

### CODE REVIEW

File: definitions/npm/prosemirror-model_v1.x.x/flow_v0.59.x-/prosemirror-model_v1.x.x.js [hunk 1]
```diff
@@ -0,0 +1,440 @@
+// flow-typed signature: 51dc3696f1c0e271adf54cc359852262
+// flow-typed version: <<STUB>>/prosemirror-model_v^1.0.0/flow_v0.57.3
```
Comment (GAntoine) [inline definitions/npm/prosemirror-model_v1.x.x/flow_v0.59.x-/prosemirror-model_v1.x.x.js]: Replace these two comments with `// @flow`.

File: definitions/package-lock.json [hunk 2]
```diff
@@ -1112,14 +1112,6 @@
           "resolved": "https://registry.npmjs.org/sprintf-js/-/sprintf-js-1.0.3.tgz",
           "integrity": "sha1-BOaSb2YolTVPPdAVIDYzuFcpfiw="
         },
-        "string_decoder": {
```
Comment (GAntoine) [inline definitions/package-lock.json]: Undo these changes to this file.

**Comment by GAntoine** [comment]
> fix tests
> actually fix tests

Story of my life. Nice work, got a couple comments on the review.

**Comment by rpowelll** [comment]
@GAntoine fixed
